### PR TITLE
Update CLI utilities libraries to pass execute module parameters 

### PR DESCRIPTION
**Pull request by vamahaja**
# Description
Fix issues from cli libraries and rm-cluster test case, this includes -

1. Remove unnecessary imports
2. Update `mgr.diable_module` to support all options with `mgr.module`
3. Align all module comments
4. Add support to pass execute module parameters to `cli.utilities.utils` module

### CODE REVIEW

**Comment by vamahaja** [comment]
Verified on RHCS 5.3.
Test Run - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-S4T15W/

**Comment by pranavprakash20** [comment]
@vamahaja since we are using keyword args with the execute command, shouldn't we check for them and append to the command here https://github.com/red-hat-storage/cephci/blob/master/cli/__init__.py#L5 ?

File: cli/ceph/ceph.py [hunk 1]
```diff
@@ -15,26 +12,17 @@ def __init__(self, nodes, base_cmd=""):
...
 
-    def status(self):
+    def status(self, **kw):
         """Get ceph status."""
         cmd = f"{self.base_cmd} status"
-        out = self.execute(sudo=True, check_ec=False, long_running=False, cmd=cmd)
-        if isinstance(out, tuple):
-            return out[0].strip()
-        return out
+        return self.execute(cmd=cmd)
```
Comment (pranavprakash20) [inline cli/ceph/ceph.py:23]: accepting a kw arg is of no use here, as we are not passing to execute command

Comment (vamahaja) [reply cli/ceph/ceph.py]: Reverted this change. Not required now.

**Comment by vamahaja** [comment]
> @vamahaja since we are using keyword args with the execute command, shouldn't we check for them and append to the command here https://github.com/red-hat-storage/cephci/blob/master/cli/__init__.py#L5 ?

Reverted changes related to this. Not required now.

**Review by psathyan — APPROVED** [review]

**Pull request merged**